db/migrate/20211014133150_drop_npq_profiles.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class DropNPQProfiles < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :npq_profiles do |t|
+      t.references :user, null: false, foreign_key: true, type: :uuid
+      t.references :npq_lead_provider, null: false, foreign_key: true, type: :uuid
+      t.references :npq_course, null: false, foreign_key: true, type: :uuid
+
+      t.date :date_of_birth
+      t.text :teacher_reference_number
+      t.boolean :teacher_reference_number_verified, default: false
+      t.text :school_urn
+      t.text :headteacher_status
+
+      t.boolean :active_alert, default: false
```

Comment:
Same. As we're doing it for the eligible_for_funding why have the possiblliity of 3 state booleans elsewhere.